packages/client/src/app/components/site-sidenav/notifications/notifications.component.ts

```diff
@@ -39,10 +43,31 @@ export class NotificationsComponent implements OnInit {
   public fetchData(): void {
     this.loading = true;
     this.notif.getAllNotifications().subscribe(data => {
-      this.unread = data.filter(val => { return val.read !== true });
+      this.unread = data.filter(val => { return val.read !== true }) as NotificationSelect[];
       this.unreadTotal = this.unread.length;
-      this.read = data.filter(val => { return val.read === true });
+      this.read = data.filter(val => { return val.read === true }) as NotificationSelect[];
+      this.loading = false;
     });
   }
 
+  selectNotif(notifId: string): void {
+    this.selectedNotifs.push(notifId);
+  }
+
+  deselectNotif(notifId: string): void {
+    this.selectedNotifs = this.selectedNotifs.filter(val => { return val !== notifId});
+  }
+
+  markAsRead(): void {
+    const request: MarkReadRequest = {
+      ids: this.selectedNotifs
+    };
+
+    this.notif.markAsRead(request).subscribe(() => {
+      this.selectedNotifs = [];
+      this.fetchData();
+    }, () => {
```

Comment:
Can this be `err => { ...` just so it's a little more immediately clear it's the error block, even if we don't do anything with `err`?

Reply:
i could tag it `_err` so it does pop up with "unused variable" warnings